osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -122,8 +135,14 @@ private double computeAimValue(ScoreInfo score, OsuDifficultyAttributes attribut
 
             if (attributes.SliderCount > 0)
             {
-                double estimateSliderEndsDropped = Math.Clamp(Math.Min(countOk + countMeh + countMiss, attributes.MaxCombo - scoreMaxCombo), 0, estimateDifficultSliders);
-                double sliderNerfFactor = (1 - attributes.SliderFactor) * Math.Pow(1 - estimateSliderEndsDropped / estimateDifficultSliders, 3) + attributes.SliderFactor;
+                double estimateSliderEndsDropped;
+                if (useClassicSlider)
+                    estimateSliderEndsDropped = Math.Clamp(Math.Min(countOk + countMeh + countMiss, attributes.MaxCombo - scoreMaxCombo), 0, estimateDifficultSliders);
+                else
+                    estimateSliderEndsDropped = Math.Min(countSliderEndsDropped + countLargeTickMiss, estimateDifficultSliders);
```

Comment:
I'm not sure `min(drops, difficultSliders)` is necessary here since we definitely know how many dropped sliders there are

Reply:
I just did it this way to match live since stable scores are clamped in the same way

Reply:
> I'm not sure `min(drops, difficultSliders)` is necessary here since we definitely know how many dropped sliders there are. It'd be better to adjust slider nerf curve instead imo

it is necessary because amount of dropped sliderends can be higher than `difficultSliders`, as `difficultSliders` is only 15% of the whole sliders count

Reply:
I'm still inclined to keep it as is since I don't want to change the curve and I *really* don't want to open the gates for negative pp values

Reply:
on second thought maybe we should change the curve

Reply:
I think keeping it as is is better for now